src/lib/libstack_trace.js

```diff
@@ -269,7 +269,7 @@ var LibraryStackTrace = {
     } else {
       name = wasmOffsetConverter.getName(pc);
     }
-    if (_emscripten_pc_get_function.ret) _free(_emscripten_pc_get_function.ret);
+    _free(_emscripten_pc_get_function.ret | 0);
```

Comment:
Why is this needed?

Reply:
Thanks, right... I keep forgetting about BigInts there.

Reply:
But it allows 0? Will it convert anything that `Number.isSafeInteger()` returns true for?